mesos.sh

```diff
@@ -33,10 +35,13 @@ rsync -av --delete --exclude="**/.git" $SOURCEDIR/ .
 ./bootstrap
 mkdir build
 cd build
+sed -i.bak -e's/c++11/c++20/' ../configure
```

Comment:
wouldn't it be safer to modify https://github.com/AliceO2Group/mesos/blob/1.11.0-alice2/configure.ac to aim at c++20 instead of 11? In principle, `c++11` strings could be present in some autogenerated if statements, but I am not experienced with autotools...

Reply:
> but I am not experienced with autotools

The fact that I am old, does not mean I am... ;-) That said, I might have found where it's specified...

Reply:
Nevermind. Apparently we do not use an autotools which allows us to check for c++20...